src/bindings/python/tests/test_runtime/test_compiled_model.py

```diff
@@ -276,3 +276,73 @@ def test_compiled_model_from_buffer_in_memory(request, tmp_path, device):
 
     compiled = core.compile_model(model=xml, weights=weights, device_name=device)
     _ = compiled([np.random.normal(size=list(input.shape)).astype(dtype=input.get_element_type().to_dtype()) for input in compiled.inputs])
+
+
+@pytest.fixture(scope="function")
+def model(dynamic_shapes):
+    partial_shape = PartialShape([1, 2048, -1]) if dynamic_shapes else PartialShape([])
+    input_shape = [Shape([1, 2048, 7])] if dynamic_shapes else [Shape([1, 2048, 7]), Shape([1, 2048, 10])]
+
+
+    max_shape_size = len(input_shape)
+    targetStaticShapes = []
+
+    if partial_shape.rank == 0:
+        partial_shape = input_shape[0]
+
+    for i in range(max_shape_size):
+        if i < len(input_shape):
+            targetStaticShapes.append(input_shape[i])
+        else:
+            targetStaticShapes.append(input_shape[-1])
+
+    param = ops.parameter(partial_shape, Type.f32)
+    relu0 = ops.relu(param)
+    print(partial_shape)
+
+    # # Convolution params
+    # kernel_3x3 = Shape([3, 3])
+    strides_1x1 = np.array([1, 1])
+    dilations_1x1 = np.array([1, 1])
+    pads_begin = np.array([0, 0])
+    pads_end = np.array([0, 0])
+    #filter_shape_1x1 = Shape([512, 1, 1, 1])
+    #filter_weights_1x1 = np.random.uniform(1, 9, filter_shape_1x1).astype(np.float32)
+
+    #conv1 = ops.convolution(relu0, ops.constant(filter_weights_1x1), strides_1x1, pads_begin, pads_end, dilations_1x1)
+    
+    filter_shape = [512, 1] + [1]
+    filter_weights = np.random.rand(*filter_shape).astype(np.float32)
+
+    # Create convolution node
+    conv = ops.convolution(relu0, filter_weights, strides_1x1, pads_begin, pads_end, dilations_1x1)
+
+    # relu1 = ops.relu(conv1)
+
+    # relu_shape = relu1.get_partial_shape()
+
+    # filter_shape_3x3 = Shape([512, filter_shape_1x1[0], 3, 3])
+    # filter_weights_3x3 = np.random.uniform(0, 1, filter_shape_3x3).astype(np.float32)
+    # conv2 = ops.convolution(relu1, ops.constant(filter_weights_3x3), strides_1x1, pads_begin, pads_end, dilations_1x1)
+    # relu2 = ops.relu(conv2)
+
+    # filter_shape_1x1_last = Shape([2048, filter_shape_3x3[0], 1, 1])
+    # filter_weights_1x1_last = np.random.uniform(0, 1, filter_shape_1x1_last).astype(np.float32)
+    # conv3 = ops.convolution(relu2, ops.constant(filter_weights_1x1_last), strides_1x1, pads_begin, pads_end, dilations_1x1)
+    # add = ops.add(conv3, relu0)
+    # reduce = ops.reduce_mean(add, np.array([2], dtype=np.int32), True)
+
+    # return Model([reduce], [param])
```

Comment:
why don't you use model with concat? I supposed some time ago we agreed to use as simple model as possible in python tests

Reply:
Yep, I've updated it